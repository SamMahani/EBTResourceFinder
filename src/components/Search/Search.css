.labelWrapper {
  padding: 1em;
  font-weight: 600;
}

.inputBar {
  outline: none;
  height: 1.5em;
}

.submitButton {
  color: var(--mainLight);
  background-color: var(--mainDark);
  margin: 1em;
  padding: 0.25em 1em;
  border-radius: 6px;
  text-align: center;
  outline: none;
  font-size: 1em;
  font-weight: 300;
}
