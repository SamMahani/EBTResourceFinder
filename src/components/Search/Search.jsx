import React from "react";

const Search = () => {
  return (
    <form>
      <label>
        Zip:
        <input type="text" name="zip" />
      </label>
      <input type="submit" value="Submit" />
    </form>
  );
};

export default Search;
