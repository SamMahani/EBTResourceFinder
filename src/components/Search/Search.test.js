import React from "react";
import { render, cleanup } from "react-testing-library";
import "jest-dom/extend-expect";
import Search from "./index.js";

afterEach(cleanup);

it("renders", () => {
  const { asFragment, getByText } = render(<Search />);
  expect(asFragment()).toMatchSnapshot();
  expect(getByText("Zip:")).toHaveClass("labelWrapper");
});
