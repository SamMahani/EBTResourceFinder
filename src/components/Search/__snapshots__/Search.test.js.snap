// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <form>
    <label
      class="labelWrapper"
    >
      Zip: 
      <input
        class="inputBar"
        name="zip"
        type="text"
        value=""
      />
    </label>
    <input
      class="submitButton"
      type="submit"
      value="Submit"
    />
  </form>
</DocumentFragment>
`;
